test2.js

```diff
@@ -1,22 +1,22 @@
 // є zякась стрінга. Треба знайти найдовшу послідовність однаокових букв
-const myString = "hh jjj dfg d f fff nn kk";
+const myString = 'hh jjj dfg d f fff nn kk';
 const getLongestSequence = (s) => {
-  let str1 = "",
-    strmax = str1;
-
-  debugger;
-  for (let ch of s) {
-    if (ch == str1[0]) {
-      str1 = str1 + ch;
-    } else {
-      if (strmax.length <= str1.length) {
+    let str1 = '',
         strmax = str1;
-      }
-      str1 = ch;
+
+    debugger;
+    for (let ch of s) {
+        if (ch == str1[0]) {
+            str1 = str1 + ch;
+        } else {
+            if (strmax.length <= str1.length) {
+                strmax = str1;
+            }
+            str1 = ch;
+        }
```

Comment:
seems to be a bit complex solution. Try to remove `str1` and `strmax` variables and have only smth like `currentLongestSequence` variable

Reply:
then firstly rename veriables - `str1` and `strmax` are not self-descriptive names

Reply:
I thought "currentLongestSequence" is a method of string.
Now is everything clear